packages/apps/stock/src/screens/inventories/InventoryLineDetailsScreen.js

```diff
@@ -30,45 +30,73 @@ import {
   ProductCardInfo,
   InventoryLineQuantityCard,
   InventoryLineButtons,
+  ProductTrackingNumberSelect,
 } from '../../components';
+import {
+  addTrackingNumber,
+  fetchInventoryLine,
+} from '../../features/inventoryLineSlice';
 import {fetchProductWithId} from '../../features/productSlice';
 import Inventory from '../../types/inventory';
 
+const trackingScanKey = 'tracking_inventory-select';
+
 const InventoryLineDetailsScreen = ({route, navigation}) => {
-  const inventory = route.params.inventory;
-  const inventoryLine = route.params.inventoryLine;
-  const product = route.params.product;
-  const trackingNumber =
-    inventoryLine != null
-      ? inventoryLine.trackingNumber
-      : route.params.trackingNumber;
+  const {inventory, inventoryLineId, productId} = route.params;
```

Comment:
As you are changing params format, you need to change it everywhere this screen is called, for example InventorySelectProductScreen 
Please check it is also the case for customer delivery and internal move as they have the same params change